### ISSUE TITLE
RelatedObjectDoesNotExist

### ISSUE THREAD

**Issue by jedisct1** [certsocietegenerale/FIR#2]
After following the dev environment setup guide, the dashboard doesn't seem to properly show up.

Calls to `/dashboard/blocked/` and `/dashboard/open/` end up with a 500 return code:

```
Environment:


Request Method: GET
Request URL: http://REDACTED:8000/dashboard/blocked/?order_by=date&asc=false&page=1

Django Version: 1.7.6
Python Version: 2.7.9
Installed Applications:
('django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.sites',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'incidents',
 'django.contrib.admin',
 'fir_plugins',
 'fir_artifacts')
Installed Middleware:
('django.middleware.common.CommonMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware')


Traceback:
File "/home/j/.local/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  111.                     response = wrapped_callback(request, *callback_args, **callback_kwargs)
File "/home/j/.local/lib/python2.7/site-packages/django/contrib/auth/decorators.py" in _wrapped_view
  21.                 return view_func(request, *args, **kwargs)
File "/home/j/.local/lib/python2.7/site-packages/django/contrib/auth/decorators.py" in _wrapped_view
  21.                 return view_func(request, *args, **kwargs)
File "/home/j/src/FIR/incidents/views.py" in dashboard_blocked
  2029. 	return incident_display(request, Q(status='B'))
File "/home/j/src/FIR/incidents/views.py" in incident_display
  1992. 		incidents_per_page = request.user.profile.incident_number
File "/home/j/.local/lib/python2.7/site-packages/django/utils/functional.py" in inner
  225.         return func(self._wrapped, *args)
File "/home/j/.local/lib/python2.7/site-packages/django/db/models/fields/related.py" in __get__
  428.                     self.related.get_accessor_name()

Exception Type: RelatedObjectDoesNotExist at /dashboard/blocked/
Exception Value: User has no profile.
```

**Comment by gaelmuller** [comment]
The 'admin' user is not meant to access incidents, but only the admin panel (at /admin). 'dev' user should not have this problem.

We will however provide a fix for this later.

**Comment by jedisct1** [comment]
The `dev` user indeed doesn't have this problem. Thanks for clarifying this!

**Comment by tomchop** [comment]
Fixed in 05587addf11e5665b0befde3472f7037af5a20e0
